### PR TITLE
Use tslib PlotImportance in optuna-dashboard

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

NA

## What does this implement/fix? Explain your changes.

This PR uses tslib's PlotImportance in optuna-dashboard

### CODE REVIEW

File: optuna_dashboard/ts/components/GraphHyperparameterImportances.tsx [hunk 1]
```diff
@@ -64,100 +72,3 @@ const GraphHyperparameterImportanceBackend: FC<{
...
-    },
-    barmode: "group",
-    bargap: 0.15,
-    bargroupgap: 0.1,
-    uirevision: "true",
-    template: colorTheme,
-    legend: {
-      x: 1.0,
-      y: 0.95,
-    },
```
Comment (porink0424) [inline optuna_dashboard/ts/components/GraphHyperparameterImportances.tsx:136]: Could you add this property to `tslib/react/src/components/PlotImportance.tsx`?

**Review by porink0424 — CHANGES_REQUESTED** [review]
Thank you for your PR! I left just one small comment.

File: optuna_dashboard/ts/components/GraphHyperparameterImportances.tsx (hunk 1 above)
Comment (keisuke-umezawa) [reply optuna_dashboard/ts/components/GraphHyperparameterImportances.tsx:136]: Added.

**Comment by keisuke-umezawa** [comment]
@porink0424 
I fixed code. Could you review it again?

**Review by porink0424 — APPROVED** [review]
LGTM!

**Pull request merged**